nano/node/scheduler/buckets.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <nano/lib/numbers.hpp>
+#include <nano/node/active_transactions.hpp>
+#include <nano/node/prioritization.hpp>
+
+#include <boost/optional.hpp>
+
+#include <condition_variable>
+#include <deque>
+#include <memory>
+#include <thread>
+
+namespace nano
+{
+class block;
+class node;
+namespace scheduler
```

Comment:
Maybe consider specifing the `nano::scheduler` namespace in a single statement (with forward declarations in separate block). Nesting namespaces introduces additional indentation, we do both in our codebase which is inconsistent. Using non-nested version makes it consistent with the most common case.